pkgs/applications/networking/instant-messengers/fluffychat/default.nix

```
@@ -21,6 +21,20 @@ flutter.mkFlutterApp rec {
     olm
   ];
 
+  flutterExtraFetchCommands = ''
+    M=$(echo $TMP/.pub-cache/hosted/pub.dartlang.org/matrix-*)
+    sed "s|/usr/lib/x86_64-linux-gnu/libolm.so.3|${olm}/lib/libolm.so.3|g" -i $M/scripts/prepare.sh
+    sed "s|if which flutter >/dev/null; then|exit; if which flutter >/dev/null; then|g" -i $M/scripts/prepare.sh
+
+    # debug
+    # cat $M/scripts/prepare.sh
+
+    pushd $M
+    chmod +x ./scripts/*.sh
+    ./scripts/prepare.sh
```

If this is only done to execute one script we can also do

```suggestion
    bash scripts/prepare.sh
```